raiden-ts/src/raiden.ts

```diff
@@ -955,6 +956,29 @@ export class Raiden {
       )
       .toPromise();
   }
+
+  /**
+   * Mints the amount of tokens of the provided token address.
+   * Throws an error if executed on main net.
+   *
+   * @param tokenAddress - Address of the token to be minted
+   * @param amount - Amount to be minted
+   * @returns transaction
+   */
+  public async mint(tokenAddress: string, amount: BigNumberish): Promise<ContractTransaction> {
+    // Check whether address is valid
+    if (!Address.is(tokenAddress)) throw new Error('Invalid address');
+
+    // Check whether we are on a test network
+    const { name } = await this.deps.provider.getNetwork();
+    if (name === 'homestead') {
+      throw new Error('Minting is only allowed on test networks.');
+    }
+
+    // Mint token
+    const customTokenContract = CustomTokenFactory.connect(tokenAddress, this.deps.signer);
+    return customTokenContract.functions.mint(decode(UInt(32), amount));
```

Comment:
Please, await on the call, then await on tx to be mined, it's useful to ensure throwing early if tx fails:
```suggestion
    const tx = await customTokenContract.functions.mint(decode(UInt(32), amount));
    const receipt = await tx.wait();
    if (!receipt.status) throw...
    return tx.hash;
```